magicblock-committor-service/src/intent_execution_manager/intent_execution_engine.rs

```diff
@@ -246,7 +246,10 @@ where
             .execute(intent.inner.clone(), persister)
             .await
             .inspect_err(|err| {
-                error!("Failed to execute BaseIntent: {:?}", err)
+                error!(
```

Comment:
This seems to be just a formatting change.
I've seen that our fmt is out of date, i.e. if I run `make ci-fmt` locally then I get a huge amount of changes.
It seems our workflow that checks `fmt` is no longer working as expected.

(Unrelated to this PR, but this change here just confirms that).

Reply:
The log message changed - formatting changed. `make ci-fmt` doesn't complain on my side